controller/controller_server/config.py

```diff
@@ -37,6 +37,7 @@
 PARAMETERS_CAPACITY_DELIMITER = "="
 PARAMETERS_CAPABILITIES_DELIMITER = "="
 PARAMETERS_OBJECT_ID_DELIMITER = ":"
+PARAMETERS_INTERNAL_ID_DELIMITER = ";"
```

Comment:
I think we now need better names to distinguish between the delimiters.
how about:
```suggestion
PARAMETERS_OBJECT_ID_INFO_DELIMITER = ":"
PARAMETERS_OBJECT_IDS_DELIMITER = ";"
```

Reply:
done